sites/platform/src/create-apps/app-reference.md

```diff
@@ -375,31 +412,122 @@ applications:
 ```
 {{% /version/specific %}}
 
-For examples of how to set up a `service` mount, see the dedicated [Network Storage page](/add-services/network-storage.md).
+### Ensure continuity when changing the name of your mount
 
-The accessibility to the web of a mounted directory depends on the [`web.locations` configuration](#web).
-Files can be all public, all private, or with different rules for different paths and file types.
+Changing the name of your mount affects the default `source_path`.
 
-Note that when you back up an environment,
-the mounts on that environment are backed up too.
+Say you have a `/web/uploads/` mount with an undefined `source_path`:
 
-Also, mounted directories aren't deleted when they're removed from `{{< vendor/configfile "app" >}}`.
-The files still exist on disk until manually removed (or, for `tmp` mounts, until the app container is moved to another host during a maintenance operation).
+```yaml {configFile="app"}
+mounts:
+    '/web/uploads/':
+        source: local
+```
+
+ If you rename the mount to `/my/uploads/`, it will point to a new, empty `/my/uploads/` directory.
+ 
+ To ensure continuity, you need to explicitly define the `source_path` as the previous name of the mount, without leading or trailing slashes:
+ 
+ ```yaml {configFile="app"}
+mounts:
+    '/my/uploads/':
+        source: local
+        source_path: web/uploads
+```
+ 
+The `/my/uploads/` mount will point to the original `/web/uploads/` directory, maintaining access to all your existing files in that directory.
 
 {{% version/only "2" %}}
 
-### Mounts, instances, and Network Storage
+### Share a mount between several apps
 
-`storage` mounts can be shared **between different instances of the same app**,
+By design, [`storage` mounts](#mounts) are shared **between different instances of the same app**,
```

Comment:
Same question as above, as `storage` mounts are only available on Upsun (and thus so are shared mounts using the mechanism described here)

Reply:
This one has a {{% version/only "2" %}} shortcode applied to it so it's only shown in the Upsun output (version 1 = PSH, version 2 = Upsun). But yeah it can be confusing, and those two pieces of feedback from you only confirm to me that the separation must be clearer, so there can be no confusion and we get what's intended in prod.